packages/ods/react/tests/_app/src/components/ods-textarea.tsx

```diff
@@ -1,19 +1,39 @@
-import React from 'react-dom/client';
+import { useFormik } from 'formik';
```

Comment:
you can remove the whole Formik testing as you're able to reproduce the loop use case on native code